open-api-spec.yaml

```diff
@@ -793,6 +900,68 @@ components:
           $ref: '#/components/schemas/amount'
         sendAmount:
           $ref: '#/components/schemas/amount'
+        description:
+          type: string
+          description: Human readable description of the outgoing payment that will be visible to the account holder and shared with the receiver.
+          example: APlusVideo subscription
+        externalRef:
+          type: string
+          description: A reference that can be used by external systems to reconcile this payment with their systems. E.g. An invoice number. (Optional)
+          example: 'customer: 847458475'
+    quote:
+      title: Quote
+      description: 'A **quote** resource represents a payment that will be, is currently being, or has previously been, sent from the account.'
+      type: object
+      examples:
+        - id: 'https://openpayments.guide/alice/quotes/ab03296b-0c8b-4776-b94e-7ee27d868d4d'
+          accountId: 'https://openpayments.guide/alice/'
+          receivingAccount: 'https://openpayments.guide/aplusvideo/'
+          receiveAmount:
+            amount: 2500
+            assetCode: USD
+            assetScale: 2
+          sendAmount:
+            amount: 2500
+            assetCode: USD
+            assetScale: 2
+          fundsAvailable: true
```

Comment:
Let's drop `fundsAvailable` as discussed. If there are no funds available then the quote should fail.